src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -254,6 +254,8 @@ class DropdownAutoCompleteMenu extends React.Component {
       emptyMessage,
       noResultsMessage,
       style,
+      rootClassName,
```

Comment:
I think you need a `propTypes` for this

Reply:
I just added a handful of proptypes right before this review!